MELA/setup.sh

```diff
@@ -16,6 +16,8 @@ getMELAARCH(){
     echo slc7_amd64_gcc820
   elif [[ "$GCCVERSION" == "8"* ]]; then
     echo slc7_amd64_gcc830
+  elif [[ "$GCCVERSION" == "12"* ]]; then
```

Comment:
Do we really need this?

Reply:
ZZGG for MCFM does not work otherwise in newer CMSSW made for el9. Is there a specific reason you are opposed to this? Are you concerned about something regarding the architectures?

Reply:
Ok, I was just checking that we are not releasing new compiled libraries for no good reason.